drivers/SmartThings/zigbee-switch/profiles/multi-switch-no-master.yml

```diff
@@ -9,10 +9,4 @@ components:
       - id: refresh
         version: 1
     categories:
-      - name: Switch
```

Comment:
Will these profile changes cause the upload to fail @greens ?

Reply:
They might. @NikitaKuchta you should be able to check with local package uploading by uploading and assigning to the same channel from main and then from this branch.